### DISCUSSION TITLE
Upper widget show mixer instead output ?

### DISCUSSION THREAD

**Discussion by MicheleVilla** [EdgeTX/edgetx#3130] [General]
In upper widget I chose "show CH1" , in this way the ch1 output is showed
In special function is set override CH1
When override CH1 is activate while the channel 1 output is frozen at the programmed value,  widget keep showing the channel that varies 
Is this behavior correct or could it be a mistake?
Thank you
EDIT
Radiomaster T16 Edgetx 2.8

**Comment by MicheleVilla** [comment]
In 2.9 the behaviour is the same
In my opinion "show CH1" should display CH1 output, that is, the actual channel position transmitted to the receiver

**Reply by pfeerick** [reply]
Need a little more information or perhaps the correct terminology?

Upper widget - do you mean you're using a widget in the top bar (i.e. as configured via Screen Settings -> User Interface (first tab) -> Top Bar -> Setup Widgets)?

If so, what widget? Do you mean you're using the "Value" widget, and setting it to show CH1 as the source? 

In that case, I would tend to agree with you, as I would have expected the CH1 sources to relate to the output channels,  not the mixer channels, but I would need to see if that is a bug, or a misunderstanding as to the purpose of the CHX sources.

**Comment by MicheleVilla** [comment]
Need a little more information or perhaps the correct terminology?

_Upper widget - do you mean you're using a widget in the top bar (i.e. as configured via Screen Settings -> User Interface (first tab) -> Top Bar -> Setup Widgets)?_
**Yes, widget in the top bar configured via Screen Settings -> User Interface (first tab) -> Top Bar -> Setup Widgets**

_If so, what widget? Do you mean you're using the "Value" widget, and setting it to show CH1 as the source?_
**Yes Value widget that show the CH1 as source( that is the CH1 value in output to receiver)**

In that case, I would tend to agree with you, as I would have expected the CH1 sources to relate to the output channels, not the mixer channels, but I would need to see if that is a bug, or a misunderstanding as to the purpose of the CHX sources.

Ok
